src/renault_api/kamereon/enums.py

```diff
@@ -0,0 +1,40 @@
+"""Kamereon enums."""
+from enum import Enum
+
+
+class EnergyCode(Enum):
+    """Enum for vehicleDetails energy code."""
+
+    ESSENCE = "ESS"
+    ELECTRIQUE = "ELEC"
```

Comment:
```suggestion
    GASOLINE = "ESS"
    ELECTRIC = "ELEC"
```

Reply:
I have removed the enum, to keep just the Renault string.
I have added uses_electricity() / uses_fuel() methods instead.